pkg/crypto/crypto.go

```diff
@@ -454,6 +476,18 @@ func GetCA(certFile, keyFile, serialFile string) (*CA, error) {
 	}, nil
 }
 
+func GetCAFromBytes(certBytes, keyBytes []byte) (*CA, error) {
+	caConfig, err := GetTLSCertificateConfigFromBytes(certBytes, keyBytes)
```

Comment:
It seems unnecessary to have both `GetCAFromBytes` and `GetTLSCertificateConfigFromBytes`. Can you simplify it before we merge?